src/Microsoft.DotNet.Darc/DarcLib/Models/Darc/DependencyGraph.cs

```diff
@@ -776,7 +776,7 @@ private static bool BuildContributesToNode(DependencyGraphNode node, Build poten
         return allCyclesRootedAtNode;
     }
 
-    private static string GetRepoPath(
+    private static async Task<string> GetRepoPath(
```

Comment:
Nit: Use the "Foo**Aysnc**" naming pattern when methods return Task.

Reply:
Changed in all touched files